ops/src/main/scala/ammonite/ops/package.scala

```diff
@@ -41,4 +41,46 @@ package object ops extends Extensions with RelPathStuff{
 
 
   implicit def fileData(p: Path) = stat.full(p)
+  
+  private val bashCompletionFunction = """__print_completions() { for ((i=0;i<${#COMPREPLY[*]};i++)); do echo ${COMPREPLY[i]}; done }"""
+  implicit class BashInterpolator(val sc: StringContext) {
```

Comment:
Well, documentation pending but: in this branch string interpolators support completion so that when you type, for example: sh"some text<TAB>", the REPL tries to invoke StringContext(texts).sh.suggestions(args).
So in this case, if you do for example: sh"git branch <TAB>" it would autocomplete with the branches, or sh"git l<TAB>" yields log.